sessions/react-nesting/navigation/README.md

```diff
@@ -0,0 +1,90 @@
+# React Nesting: Navigation
+
+For now, there is a simple navigation in the [`src/App.js`](./src/App.js) file which only uses built-in React components.
+
+Let's refactor to use custom components instead; you will encounter the `children` prop and recap props in general.
+
+## Task
+
+In order to provide a clear guide, the exercise is split into sub-tasks to help you following along.
+
+### 1. Header component
+
+Task: Outsource the `<header>` into a `Header` component.
+
+- In the `/src` folder, create a subfolder called `components`. This is where all component files will live.
+- In `/src/components`, create a file called `Header.js`.
+  - Export a `Header` component which returns the same `<header></header>` element as in the `App.js` file.
+  - Don't forget the `className` attribute.
+  - The function should receive a `children` prop and render it between the opening and closing `<header>` tags.
+- In the `App.js` file, import the `Header` component and replace the `<header></header>` tags with it.
+
+🎉 Congratulations, you've just created a wrapping `Header` component using the `children` prop!
+
+### 2. Navigation component
+
+Let's do (almost) the same with the navigation.
+
+Task: Outsource the `<nav>` into a `Navigation` component.
+
+- Create `src/components/Navigation.js`.
+- The `Navigation` should receive a `children` prop used in its return statement.
+  - 💡 Note that the `nav` does not have a `className`, so there is no need to add this attribute.
+- In the `App.js` file, import the `Navigation` component and replace the `<nav></nav>` tags with it.
+
+Understood the `children` prop now? Let's dive deeper!
+
+### 3. Link component
+
+Task: Create a `Link` component to render `<a>` tags.
+
+- Create a file for the `Link` component.
+- The `Link` component should receive the props
+  - `href` which is passed to the `href` attribute of the `<a>` tag and
+  - `children` passed between the opening and closing `<a>` tags.
+- Don't forget the `className` attribute.
```

Comment:
This can be interpreted in two ways:

1. The `className` is hardcoded within the component
2. The `className` is a prop

I was tempted to do two to replace all links in the app… this should be a little more clear